content/en/developers/metrics/_index.md

```diff
@@ -17,20 +18,28 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric (with [some exception][2]).
+
+A custom metric refers to **a unique combination of metric name, host, and tag values**. In general, any metric you send using [DogStatsD][3], or through a [custom Agent Check][4] is a custom metric. There are no enforced fixed rate limits on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][5].
+
+**Note**: Administrative users (those with [Datadog Admin roles][6] can see the total custom metrics per hour and the top 500 custom metrics for their account in the [usage details page][7]. Learn more about [how Custom metrics are counted][5].
+
+## Metric definition
+
+A Datadog metric is defined by the properties below. Refer to the [Metrics Introductionumentation][8] to learn how to graph metrics within Datadog.
 
 | Property         | Description                                                                                                                                               |
-|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `<METRIC_NAME>`  | The [name of your metric](#naming-metrics).                                                                                                               |
 | `<METRIC_VALUE>` | The value of your metric.                                                                                                                                 |
-| `<TIMESTAMP>`     | The timestamp associated with the metric value. **Note**: Metric timestamps cannot be more than 10 minutes in the future or more than 1 hour in the past. |
+| `<TIMESTAMP>`    | The timestamp associated with the metric value. **Note**: Metric timestamps cannot be more than 10 minutes in the future or more than 1 hour in the past. |
 | `<CONTEXT>`      | The set of tags associated with your metric.                                                                                                              |
```

Comment:
We definitely cannot use this "CONTEXT" terminology on public documentation from what I've been told. I'd remove this entire table quite honestly; we've already defined a custom metric above in bold and adding this table adds to confusion/contradiction as to why it's not defined by all of the properties listed in this table

Reply:
I change the tile of this section to avoid any confusion.